docs/release-notes.md

```diff
@@ -2,6 +2,36 @@
 title: Release Notes
 ---
 
+## v2.1.0
+
+### New Features
+
+- Increase scalability for [multi-gateway scenarios](./user-guide/scalability.md).
+  This eliminates several disadvantages in a [multi-LNS deployment scenario](./user-guide/deployment-scenarios.md).
+  See [ADR - 010. LNS sticky affinity over multiple sessions](./adr/010_lns_affinity.md).
+- [Standalone discovery service](./user-guide/lns-discovery.md) for dynamic LNS discovery.
+  See [ADR - 009. LoRaWAN Network Server (LNS) discovery](./adr/009_discovery.md).
+- LoRaBasicsStationModule updated to Basics Station v2.0.6:
```

Comment:
```suggestion
- LoRaBasicsStationModule v2.1.0 includes:
  - Binaries for Basics Station v2.0.6:
```

Let's try to clarify the confusion brought by mismatch in versioning

@spygi, does this make the situation better?